lib/elixir/lib/task.ex

```diff
@@ -233,8 +233,8 @@ defmodule Task do
     * `:owner` - the PID of the process that started the task
 
   """
-  @enforce_keys [:pid, :ref, :owner]
-  defstruct pid: nil, ref: nil, owner: nil
+  @enforce_keys [:pid, :ref, :owner, :mfa]
+  defstruct [:pid, :ref, :owner, :mfa]
```

Comment:
Little trick that @lexmag was a fan of many years ago:

```suggestion
  defstruct @enforce_keys
```

😛

Reply:
Sourcery!